diff --git a/dist/index.mjs b/dist/index.mjs
index 7c1c4440c6ed22a3829f828364b06669c56b31c9..f464611b5b98c2935c2d92641dfe628f161ebca4 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -240,9 +240,10 @@ class Option {
       return camelcaseOptionName(name);
     }).sort((a, b) => a.length > b.length ? 1 : -1);
     this.name = this.names[this.names.length - 1];
-    if (this.negated && this.config.default == null) {
-      this.config.default = true;
-    }
+    // disable automatic setting of the defaults for negated options
+    // if (this.negated && this.config.default == null) {
+    //   this.config.default = true;
+    // }
     if (rawName.includes("<")) {
       this.required = true;
     } else if (rawName.includes("[")) {